migrations/20221031094617-enhance-activities-links-collective-created.js

```diff
@@ -0,0 +1,42 @@
+'use strict';
+
+/** @type {import('sequelize-cli').Migration} */
+module.exports = {
+  async up(queryInterface) {
+    console.time('Linking for COLLECTIVE_CREATED activities');
+    await queryInterface.sequelize.query(`
+      UPDATE "Activities" a
+      SET "HostCollectiveId" = COALESCE(a."HostCollectiveId", a."CollectiveId"),
+          "CollectiveId" = (a.data -> 'collective' ->> 'id')::int
+      WHERE a.type = 'collective.created'
+      AND a.data -> 'collective' ->> 'id' IS NOT NULL
+    `);
+    console.timeEnd('Linking for COLLECTIVE_CREATED activities');
+
+    console.time('Linking for COLLECTIVE_APPROVED activities');
+    await queryInterface.sequelize.query(`
+      UPDATE "Activities" a
+      SET "HostCollectiveId" = COALESCE(a."HostCollectiveId", a."CollectiveId"),
+          "CollectiveId" = (a.data -> 'collective' ->> 'id')::int
+      WHERE a.type = 'collective.approved'
+      AND a.data -> 'collective' ->> 'id' IS NOT NULL
+    `);
+    console.timeEnd('Linking for COLLECTIVE_APPROVED activities');
+
+    console.time('Linking for COLLECTIVE_REJECTED activities');
+    await queryInterface.sequelize.query(`
+      UPDATE "Activities" a
+      SET "HostCollectiveId" = COALESCE(a."HostCollectiveId", a."CollectiveId"),
+          "CollectiveId" = (a.data -> 'collective' ->> 'id')::int
+      WHERE a.type = 'collective.rejected'
+      AND a.data -> 'collective' ->> 'id' IS NOT NULL
+    `);
+    console.timeEnd('Linking for COLLECTIVE_REJECTED activities');
+  },
+
+  async down() {
+    /**
+     * No rollback
+     */
+  },
```

Comment:
@Betree : I have been thinking, it's nice to add a lot here so that the user knows when they execute for example `npm run db:migrate:undo` that there's no rollback. Otherwise in the console it looks like it was rolled back. Helps later so that looking at the console log one can know which migrations rolled back and which ones did not. 🤔

Reply:
@SudharakaP I like the idea! Something like this I suppose:

```suggestion
  async down() {
    console.log('Not rolling back the changes');
  },
```

For this one I'm gonna add a real rollback query, but I'll adapt this practice on new PRs from now on

Reply:
Yes, that's what I meant. 👍🏽